### PR TITLE
Rebalances Blood Cult Stun

### PR DESCRIPTION
**Pull request by YakumoChen**
gameplay change time

## About The Pull Request
Changes the blood cult stun to no longer apply an instant hardstun (pre-change it applied 140 stamina damage). 
It instead applies 80 stamina damage on hit. 
All other effects remain identical (knocks you down, silences you, applies a long cult stutter, flashes you, etc)
Stun now has three charges and does 5 brute to yourself per use.

## Why It's Good For The Game
Cult is pretty cheesy for its instant I win button in combat. Yes, they need the stun to convert people. Applying the second hit should not be difficult in most non-combat scenarios. 
With the change, it's no longer basically impossible to deal with cult in melee. You have a much more reasonable chance of applying a stun baton or getting a shot off if you were holding a gun and got hit.

Arguably a nerf to the spell, but on the other hand, you get three uses out of it per cast now. That's enough for one and a half hard stuns.

## Changelog
:cl:
balance: Blood Cult stun now does 80 stamina damage from 140, gains three charges, and deals 5 brute damage per use.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/blood_magic.dm [hunk 1]
```diff
@@ -440,7 +441,7 @@
 		else
 			to_chat(user, "<span class='cultitalic'>In an brilliant flash of red, [L] falls to the ground!</span>")
 			L.Knockdown(160)
```
Comment (Trilbyspaceclone) [inline code/modules/antagonists/cult/blood_magic.dm]: this is a 1.6 second hardstun + stam crit do to KD adding into stam loss

**Comment by YakumoChen** [comment]
I think it works

**Comment by ghost** [comment]
this doesn't actually... really do anything though
if they click on you once you're already fucked, now they just have to click a second time while you're down to stamcrit you... so cult is still just as powerful

**Comment by YakumoChen** [comment]
It doesn't disarm, leaving opportunity for retaliation. Yes, getting the second hit in is easy if the target wasnt expecting it and you aren't in the middle of a larger fight. That is still intended.

**Comment by ghost** [comment]
idk i'm down for cult nerfs and all but this doesn't really feel like much of a nerf or a buff and just a needless change. maybe something like making a forced cooldown on how often you can stun normally, or if you're feeling like it's fun time having spells draw from your blood instead of dealing damage (and making it so blood rites puts blood in your bloodstream first before adding charges) could be cool and make it so you can't just quadstun and run around stunning people while also encouraging the use of blood rites for its healing instead of just blood beam/halberd.

**Comment by YakumoChen** [comment]
Please do not make suggestions or opinions irrelevant to this PR.

Its neither a buff or a nerf, you're already correct. It exchanges an instant hard stun for added utility.

**Comment by silicons** [comment]
not touching this for now hmm

**Comment by Ghommie** [comment]
Gonna slap a testmerge candidate label onto it.
If people don't like this PR, I'll close it, cause I don't like it either.

**Comment by DeltaFire15** [comment]
I recommend not testmerging this as it still effectively doubles mute / stutter time as both use a += as of now

**Comment by YakumoChen** [comment]
The PR no longer functions because of the clock cult stuff and some edits to how the stun works so I might remake it, likely won't

**Pull request closed without merging**